packages/cli/src/commands/install/askForPackageManager.js

```diff
@@ -0,0 +1,17 @@
+const inquirer = require('inquirer')
+
+module.exports.askForPackageManager = async () => {
+    const prompt = inquirer.createPromptModule()
+    const userInput = await prompt([
+        {
+            type: 'list',
+            name: 'packageManager',
+            message:
+                'This step will install the @dhis2/cypress-commands package. Choose your package manager',
+            choices: ['yarn', 'npm', 'skip'],
+            default: 0,
```

Comment:
Could the default option be determined based on the existence of a `yarn.lock` or a `package-lock.json` file?

If a project has neither, or both, we could present `skip` as default.

Reply:
Implemented